src/screens/Wallet.js

```
@@ -89,46 +89,24 @@ function Wallet() {
   // Initialize the screen on mount
   useEffect(() => {
     initializeWalletScreen();
-  }, []);
+  }, [initializeWalletScreen]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)